src/bin/casc/args.rs

```diff
@@ -20,8 +20,11 @@ pub struct Args {
     #[clap(default_value = "out.cil", short, value_parser = clap::builder::ValueParser::new(parse_out_filename))]
     pub out_filename: String,
     /// Build the systems from the SYSTEM_NAMES list. "-s all" to build all defined systems.
-    #[clap(short, multiple_values = true, conflicts_with = "out-filename")]
+    #[clap(short, conflicts_with = "out-filename")]
     pub system_names: Vec<String>,
+    ///colorize the output.  WHEN can be 'always', 'auto' (default), or 'never'
```

Comment:
When running casc --help the output for the color arg is indented further than the others.

Reply:
Interesting, thanks for pointing this out.

It looks to me like what is happening is that clap is aligning short args and long args.  Since --color doesn't have a short form, it's aligned to the long form of the other args.

I agree that this looks a little weird, but it's in line with (eg) rustc, which I imagine is a comparison point for the clap developers.

In a quick googling, I don't see any way that we have control over this.  It seems to be a clap design choice in how they display the --help string.  Things may possibly change when we migrate to clap 4.